odc/stats/plugins/tcw_percentiles.py

```diff
@@ -23,16 +23,43 @@ class StatsTCWPC(StatsPluginInterface):
     def __init__(
         self,
         coefficients: Dict[str, Dict[str, float]] = {
-            "wet": {"blue": 0.0315, "green": 0.2021, "red": 0.3102, "nir": 0.1594,
-                    "swir1": -0.6806, "swir2": -0.6109},
-            "bright": {"blue": 0.2043, "green": 0.4158, "red": 0.5524, "nir": 0.5741,
-                       "swir1": 0.3124, "swir2": 0.2303},
-            "green": {"blue": -0.1603, "green": -0.2819, "red": -0.4934, "nir": 0.7940,
-                      "swir1": -0.0002, "swir2": -0.1446},
+            "wet": {
+                "blue": 0.0315,
+                "green": 0.2021,
+                "red": 0.3102,
+                "nir": 0.1594,
+                "swir1": -0.6806,
+                "swir2": -0.6109,
             },
-        input_bands: Sequence[str] = ["blue", "green", "red", "nir", "swir1", "swir2", "fmask", "nbart_contiguity"],
+            "bright": {
+                "blue": 0.2043,
+                "green": 0.4158,
+                "red": 0.5524,
+                "nir": 0.5741,
+                "swir1": 0.3124,
+                "swir2": 0.2303,
+            },
+            "green": {
+                "blue": -0.1603,
+                "green": -0.2819,
+                "red": -0.4934,
+                "nir": 0.7940,
+                "swir1": -0.0002,
+                "swir2": -0.1446,
+            },
+        },
+        input_bands: Sequence[str] = [
+            "blue",
+            "green",
+            "red",
+            "nir",
+            "swir1",
+            "swir2",
+            "fmask",
+            "nbart_contiguity",
+        ],
```

Comment:
I think we should change the spectral band names to have the "nbart" prefix to match the ARD and GeoMAD bands.

Reply:
What plugin generates this: 
<img width="1482" alt="image" src="https://user-images.githubusercontent.com/65524781/175197619-e52e108e-9b77-4a61-bfa2-ecf2a2401318.png">

Reply:
this plugin.

Reply:
This looks good.